packages/react-native/scripts/cocoapods/jsengine.rb

```diff
@@ -10,9 +10,11 @@
 # @parameter react_native_path: relative path to react-native
 # @parameter fabric_enabled: whether Fabirc is enabled
 def setup_jsc!(react_native_path: "../node_modules/react-native", fabric_enabled: false)
-    pod 'React-jsc', :path => "#{react_native_path}/ReactCommon/jsc"
-    if fabric_enabled
-        pod 'React-jsc/Fabric', :path => "#{react_native_path}/ReactCommon/jsc"
+    if ENV['USE_THIRD_PARTY_JSC'] != '1'
+        pod 'React-jsc', :path => "#{react_native_path}/ReactCommon/jsc"
+        if fabric_enabled
+            pod 'React-jsc/Fabric', :path => "#{react_native_path}/ReactCommon/jsc"
```

Comment:
not even sure that this subspec is actually needed...

Reply:
maybe not but that feels out of scope toward this pr. maybe we can update that in a separate pr?

Reply:
I can create a separate one with removal